srcpkgs/Uranium/patches/disable_code_style_test.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/cmake/UraniumTests.cmake b/cmake/UraniumTests.cmake
+index 332b24c0..3c090736 100644
+--- cmake/UraniumTests.cmake
++++ cmake/UraniumTests.cmake
+@@ -50,7 +50,7 @@ foreach(_plugin ${_plugins})
+ endforeach()
+ 
+ #Add code style test.
+-add_test(
+-    NAME "code-style"
+-    COMMAND ${PYTHON_EXECUTABLE} run_mypy.py WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+-)
+\ No newline at end of file
++#add_test(
++#    NAME "code-style"
++#    COMMAND ${PYTHON_EXECUTABLE} run_mypy.py WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
++#)
```

Comment:
Typically you would just remove the test entirely, rather than patching it to be commented out.